docs/qbasepileup/index.md

```diff
@@ -1,4 +1,8 @@
 # qbasepileup
+```diff
+-This tool is dated due to some input file format (dccq, dcc1) is no longer available!
```

Comment:
dated or deprecated?

Reply:
I am waiting for JP's decision. the indel and compound SNP modes are deprecated but the SNP and coverage mode still work OK.